src/resolvers/user.ts

```diff
@@ -10,6 +10,7 @@ import isE2E from '../utils/isE2E';
 import { dateFromObjectId } from '../utils/dates';
 import { UserDBScheme } from '@hawk.so/types';
 import * as telegram from '../utils/telegram';
+import {MongoError} from "mongodb";
```

Comment:
Did you run ESLint?

Reply:
I will fix all eslint issues in separate request